reference/customers.sf.yml

```diff
@@ -57,7 +58,275 @@ paths:
                     - fieldId: '25'
                       fieldValue: Leave in backyard
         description: Data sent the the customer endpoint when creating a customer during checkout.
+  '/customers/{customerId}/metafields':
+    get:
+      summary: Get customer metafields.
+      tags:
+        - Customer Metafields
+      description: Get a customer's metafields.
+      operationId: getCustomersMetafields
+      parameters:
+        - $ref: '#/components/parameters/customerId'
+        - name: customerId
+          in: query
+          schema:
+            type: integer
+      requestBody:
+        required: true
+        content: 
+          application/json:
+            schema:
+             $ref: '#/components/schemas/metafield'
+      responses:
+        '200':
+          description: |
+            List of `Metafield` objects.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/responses/MetafieldResponse_Post'
+        '500':
+          description: Internal Server Error
+    post:
+      summary: Create Customer Metafields
+      tags:
+      - Customer Metafields
+      description: Creates Customer metafields.
+      operationId: createCustomerMetafields
+      parameters:
+        - $ref: '#/components/parameters/customerId'
+        - name: customerId
+          in: query
+          schema:
+            type: integer
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              items:
+                $ref: '#/components/schemas/MetafieldBase_Post'
+      responses:
+        '200':
```

Comment:
for post we also have 422 in case some of the entries are incorrect (or all) and we can also have Internal Server Error

Reply:
Thanks! Should I add 422 and 500 responses to all the endpoints or just the POST?

Reply:
422 and 500 should be in all batch endpoints in v3 admin, but if this just just storefront documentation I'm not aware of any work on REST batch endpoints for metafields on storefront.
@zvuki was such a thing done in storefront?